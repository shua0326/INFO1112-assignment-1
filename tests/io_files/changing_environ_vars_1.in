echo hi this is the default prompt
var PROMPT "new prompt!!$ " 
echo something is different
echo running ls to find .py files !
ls .. | grep ".py$"
echo path will be empty now
var PATH ""
ls ..
