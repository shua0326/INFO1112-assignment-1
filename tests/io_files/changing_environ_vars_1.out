>> hi this is the default prompt
>> new prompt!!$ something is different
new prompt!!$ running ls to find .py files !
new prompt!!$ mysh.py
parsing.py
new prompt!!$ path will be empty now
new prompt!!$ new prompt!!$ mysh: command not found: ls
new prompt!!$ 
